apps/app/scripts/reset-project.js

```
@@ -6,15 +6,15 @@
  * You can remove the `reset-project` script from package.json and safely delete this file after running it.
  */
 
-const fs = require('fs')
-const path = require('path')
-const readline = require('readline')
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)